FCCee/CLD/compact/CLD_o3_v01/InnerTracker_o2_v07.xml

```diff
@@ -1,34 +1,34 @@
 <lccdd>
     <define>
 
-        <constant name="InnerTracker_Barrel_radius_0" value="127*mm"/>
-        <constant name="InnerTracker_Barrel_radius_1" value="400*mm"/>
-        <constant name="InnerTracker_Barrel_radius_2" value="670*mm"/>
-
-        <constant name="InnerTracker_Barrel_half_length_0" value="481.6*mm"/>
-        <constant name="InnerTracker_Barrel_half_length_1" value="481.6*mm"/>
-        <constant name="InnerTracker_Barrel_half_length_2" value="692.3*mm"/>
-
-        <constant name="InnerTracker_Endcap_z_0" value="524*mm"/>
-        <constant name="InnerTracker_Endcap_z_1" value="808*mm"/>
-        <constant name="InnerTracker_Endcap_z_2" value="1093*mm"/>
-        <constant name="InnerTracker_Endcap_z_3" value="1377*mm"/>
-        <constant name="InnerTracker_Endcap_z_4" value="1661*mm"/>
-        <constant name="InnerTracker_Endcap_z_5" value="1946*mm"/>
-        <constant name="InnerTracker_Endcap_z_6" value="2190*mm"/>
-
-        <constant name="InnerTracker_Endcap_radius_0" value="79.5*mm"/>
-        <constant name="InnerTracker_Endcap_radius_1" value="123.5*mm"/>
-        <constant name="InnerTracker_Endcap_radius_2" value="165*mm"/>
-        <constant name="InnerTracker_Endcap_radius_3" value="207.5*mm"/>
-        <constant name="InnerTracker_Endcap_radius_4" value="249.5*mm"/>
-        <constant name="InnerTracker_Endcap_radius_5" value="293*mm"/>
-        <constant name="InnerTracker_Endcap_radius_6" value="330*mm"/>
-
-        <constant name="VertexCoolingGap"            value="5*mm"/>
-        <constant name="VertexCoolingShellThickness" value="1*mm"/>
-
-        <constant name="ITM" value="15.1*mm"/>
+        <constant name="InnerTracker_Barrel_radius_0" value="127*mm" />
+        <constant name="InnerTracker_Barrel_radius_1" value="400*mm" />
+        <constant name="InnerTracker_Barrel_radius_2" value="670*mm" />
+
+        <constant name="InnerTracker_Barrel_half_length_0" value="481.6*mm" />
+        <constant name="InnerTracker_Barrel_half_length_1" value="481.6*mm" />
+        <constant name="InnerTracker_Barrel_half_length_2" value="692.3*mm" />
+
+        <constant name="InnerTracker_Endcap_z_0" value="524*mm" />
+        <constant name="InnerTracker_Endcap_z_1" value="773*mm" />
```

Comment:
Along with the z position change we would need to adapt the inner radius accordingly to stay close to the 150 mrad coverage?

Reply:
thank you very much for spotting the issue!

Reply:
I have changed the IT endcap radius, but I am not sure if I should adapt something else, it is difficult to see by eye